src/requests/components/Loader.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+export const Loader = styled.div`
+    animation: fadeInOut 2s infinite;
+
+    @keyframes fadeInOut {
+        0% {opacity: 1};
+        50% {opacity: 0};
+        100% {opacity: 1};
+    }
+`; 
```

Comment:
new line at the end of the file - maybe You should turn on this option in Your IDE?

Reply:
in linter ;)

Reply:
I'll we can apply starter lint config in monitor 🤔